specs/DesignDiscussions/ImagevColumn.md

```diff
@@ -0,0 +1,65 @@
+# Consistency spec: Image size vs Column size
+
+## Table of Contents
+- [Overview](#overview)
+    - [Current Problem](#current-problem)
+    - [What's happening behind the scenes?](#whats-happening-behind-the-scenes)
+    - [Proposed Solutions](#proposed-solutions)
+        - [Honoring Image Size](#honoring-image-size)
+
+## Overview
+
+### Current problem
+
+As part of the solution to the Column width behaviour it was thought that images could be used as a reference to provide the minimum size needed to render an `auto` width column. This asseveration is not correct, as some of our scenario cards don't follow that principle, take for example the [FlightDetails card](), in this card we have an image whose `width` is `1000px`, if we were to honour the measurements given by the card author we would get a card where this long image occupies most of the width of the column set as can be seen in the views generated by the iOS and the Android renderers.
+
+| Platform | Rendered card |
+| --- | --- |
+| JS (Best looking rendering) | <img src="assets/ImageColumn/GoodFlightDetails.png" width="400px">| 
+| iOS | <img src="assets/ImageColumn/iOSFlightDetails.png" width="400px"> |
+| Android | <img src="assets/ImageColumn/androidFlightDetails.png" width="400px"> |
+
+### What's happening behind the scenes?
+
+As we have seen, Javascript is generating the better looking card, but how does it do it? As we can see in the columnset, there are other columns that have images on them so why don't they shrink as well? The difference between columns 1, 3, 4 and column 2 is that the second column has `stretch` width, so after changing that width to `auto` we got a similar behaviour to the iOS and Android platforms: 
+
+| Rendered view | Schema
+| --- | --- |
+| <img src="assets/ImageColumn/BadFlightDetails.png" width="800px"> | <img src="assets/ImageColumn/BadRenderSchema.png" width="800px"> |
```

Comment:
This results in a 404.. Does this need to include the relative path here vis "..\assets"?